### PR TITLE
Add favorites and playlists

### PR DESCRIPTION
**Pull request by redsolver**

### CODE REVIEW

File: src/constants/playlists.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 module.exports = [
-  {name: 'favorites', author: 'You!', title: 'Your Saved Favorites'},
```
Comment (dmarcos) [inline src/constants/playlists.js:2]: It seems there are still some formatting isssues. See space around `{`

File: src/lib/convert-beatmap.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 module.exports = function convertBeatmap (src) {
   if (src.converted) return src;
 
+  if (src['map']) {
+    src = src['map'];
```
Comment (dmarcos) [inline src/lib/convert-beatmap.js:5]: Can just assigj? `src = src['map'];`

File: src/constants/playlists.js (hunk 1 above)
Comment (redsolver) [reply src/constants/playlists.js:2]: there are spaces around the newly added lines, that's consistent with semistandard

Comment (dmarcos) [reply src/constants/playlists.js:2]: Objects should be `{name: 'favorites', author: 'You!', title: 'Your Saved Favorites'}` no space before / after curly braces

File: src/lib/convert-beatmap.js (hunk 2 above)
Comment (dmarcos) [reply src/lib/convert-beatmap.js:5]: Alternatively. `src = src['map'] || src;`

**Pull request merged**